### DISCUSSION TITLE
Question - re multiple pinned images?

### DISCUSSION THREAD

**Discussion by gavinsboyle** [sly9/VoyagerTelegramBot#29] [Q&A] [answered]
currently there is a pinned image of the stats graphs in the telegram feed.
it looks to work really well

in my studies i usually have 2 sequence files/targets run per evening.  the pinned stats image shows the latest graph of the current sequence - and at that point i appear to have lost the graph of the previous sequence, so when i wake up i can only see the performance of the run that is close to finishing off the evening and not the run that started hours before.

is it in the nature of the pinned image that only one image can be pinned or is it possible that i can set/configure for a pinned image to be there for each sequence (there would only be a couple of these each night in my case)

**Comment by bigpizza** [comment, marked as answer]
Hi Gavin,

Thanks for the suggestion. Currently we only support single pinned image. Techenically it is possible to have multiple pinned message in Telegram but we haven't got the time to actually implement the feature.

Currently we are working on the new GUI improvement and plan to revisit telegram message part in the near future. We will announce this feature when ready 😄 

Clear Skies!

Kun
